Makefile

```
@@ -11,13 +11,96 @@ PYTHON_REGISTRY = pypi
+LLAMA_STACK_PORT ?= 8321
+CONTAINER_RUNTIME ?= $(shell command -v podman 2>/dev/null || command -v docker 2>/dev/null)
+
+run: ensure-llama-stack-container ## Run the service locally with llama-stack container
+	@echo "Starting Lightspeed Core Stack..."
 	uv run src/lightspeed_stack.py -c $(CONFIG)
 
-run-llama-stack: ## Start Llama Stack with enriched config (for local service mode)
-	uv run src/llama_stack_configuration.py -c $(CONFIG) -i $(LLAMA_STACK_CONFIG) -o $(LLAMA_STACK_CONFIG) && \
-	AZURE_API_KEY=$$(grep '^AZURE_API_KEY=' .env | cut -d'=' -f2-) \
-	uv run llama stack run $(LLAMA_STACK_CONFIG)
+ensure-llama-stack-container: stop-llama-stack-container build-llama-stack-image start-llama-stack-container
```

checkmake (0.3.2)

[warning] Target "ensure-llama-stack-container" should be declared PHONY.

(phonydeclared)